clang/include/clang/Format/Format.h

```diff
@@ -922,8 +922,23 @@ struct FormatStyle {
     ///   };
     ///   int f();
     ///   int f() { return 1; }
+    ///   int foooooooooooooooooooooooooooooooooooooooooooooooo::
+    ///       baaaaaaaaaaaaaaaaaaaaar();
     /// \endcode
     RTBS_None,
+    /// Break after return type automatically, while allowing a break after
+    /// short return types.
```

Comment:
This should be rephrased, I know what the option means, but I couldn't figure it out of the description.

Reply:
I made another attempt at being clearer.
What do you think?

Reply:
Better.